### PR TITLE
Add compatibility table for grammar functionalities

### PR DESCRIPTION
**Pull request by Zorin95670**
Add compatibility table for grammar functionalities, to inform the user which functionality is available on this plugin.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,6 +24,424 @@ Plugin for managing GithubAction files in [Leto-Modelizer](https://github.com/di
...
+      <td colspan="3">public</td>
+      <td align="center">$\textcolor{red}{\textbf{\textsf{X}}}$</td>
+    </tr>
+    <tr>
+      <td width="5px" padding="0"></td><td width="5px" padding="0"></td>
+      <td colspan="3">pull_request</td>
+      <td align="center">$\textcolor{orange}{\textsf{〜}}$</td>
+    </tr>
+    <tr>
+      <td width="5px" padding="0"></td><td width="5px" padding="0"></td><td width="5px" padding="0"></td>
```
Comment (k-y0u) [inline README.md:178]: divide (for under repetition too)

File: README.md [hunk 2]
```diff
@@ -53,7 +472,8 @@ Steps to release:
 
 ### Git: Default branch
 
-The default branch is main. Direct commit on it is forbidden. The only way to update the application is through pull request.
+The default branch is main. Direct commit on it is forbidden. The only way to update the application is through pull
```
Comment (k-y0u) [inline README.md:475]: divide at `. The only`

**Review by k-y0u — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Zorin95670) [reply README.md:178]: no sorry, is more easy to edit it like that.

File: README.md (hunk 2 above)
Comment (Zorin95670) [reply README.md]: done

**Review by k-y0u — APPROVED** [review]

**Pull request merged**